ipn/ipnlocal/local.go

```diff
@@ -1960,9 +1966,10 @@ func (b *LocalBackend) WatchNotifications(ctx context.Context, mask ipn.NotifyWa
 	var ini *ipn.Notify
 
 	b.mu.Lock()
+
 	const initialBits = ipn.NotifyInitialState | ipn.NotifyInitialPrefs | ipn.NotifyInitialNetMap
 	if mask&initialBits != 0 {
-		ini = &ipn.Notify{Version: version.Long()}
+		ini = &ipn.Notify{Version: version.Long(), SessionID: sessionID}
```

Comment:
let's break this up to one field per line now (so four lines total out of this line)

Reply:
maybe we need a new NotifyWatchOpt bit to request a session ID and send it at start so they don't need InitialState? But I guess InitialState is cheap enough. Just document on the NotifyWatchOpt that setting InitialState also gets you a SessionID?